app/harvesters/uva_harvester.rb

```diff
@@ -0,0 +1,237 @@
+# For the UVA harvester,
+# it should be possible to have the harvester take multiple collection PIDs
+# as an parameter rather than requiring separate invocations for each collection.
+
+##
+# A harvester implementation for UVA
+#
+class UVAHarvester
+  include Krikri::Harvester
+
+  DEFAULT_URI_TEMPLATE =
+    'http://fedoraproxy.lib.virginia.edu/fedora/objects/uva-lib%3A[PID]/methods/uva-lib%3AmetsSDef/getMETS'
```

Comment:
Can this be changed so the template doesn't expect `uva-lib:` as a prefix, so we can pass PIDs like `uva-lib:2190487` directly to it?